### PR TITLE
sys-apps/dracut: 059-r6 revbump for systemd 255 incompatibility

### PR DESCRIPTION
**Pull request by ferringb**
changes:
- fix pcr{phrase -> extend} rename and reduction. Renamed patch isn't yet merged, PR is at: https://github.com/dracutdevs/dracut/pull/2586
- add systemd-executor (systemd flat out fails without it) This was already merged upstream.

At least for my setup, this is enough to get systemd-255/dracut behaving again.

### CODE REVIEW

**Comment by ferringb** [comment]
Hold on merging this until I update the change.  The pcrphase patch I posted works, but the version that will land in dracut is slightly different.  Same result, just syntactic differences.  See https://github.com/dracutdevs/dracut/pull/2586/ .

Once that lands, I'll refresh this PR.

**Comment by Flowdalic** [comment]
Please also add a "Closes: https://bugs.gentoo.org/919766" git trailer.

Thanks for backporting the fixes!

**Review by floppym — APPROVED** [review]
Thanks, this can be merged once @ferringb  is happy with the patch.

**Comment by ferringb** [comment]
@Flowdalic added.  @floppym go nuts for the merge, w/ the caveat that upstream hasn't yet merged my [fix](https://github.com/dracutdevs/dracut/pull/2586), but I expect it to be merged as is.

**Pull request closed without merging**
